migration/src/main/java/uk/ac/bbsrc/tgac/miso/migration/MigrationData.java

```diff
@@ -104,4 +106,17 @@ public void setBoxes(Collection<Box> boxes) {
     this.boxes = boxes;
   }
 
+  /**
+   * @return all box position data. The key of the outer Map is the box alias. The key of the inner Map
+   *         is the box position. Note: Saving these will likely that the Boxables have been saved first
```

Comment:
Is this note missing a verb after `likely`?

Reply:
Deleted wrong word

Reply:
fixed